# Ejercicios

#### 1 Ejercicio

Para calcular el importe de IVA de  145 Euros es necesario multiplicar 145 por 0.21.
Escribe en Python la expresión que calcula el IVA de 145 Euros.

In [2]:
# Sol:

# El resultado es 30.45

Escribe en Python la expresión que calcula el IVA de 258 Euros.

In [2]:
# Sol:

258 * 0.21

54.18

Escribe una función en Python llamada __calcula_iva__ que calcule el importe de IVA de una cierta cantidad $x$.

In [3]:
# Sol

def calcula_iva(x):
    i = x * 0.21
    return(i)

In [4]:
calcula_iva(9999)

2099.79

In [5]:
calcula_iva(145)

30.45

In [6]:
calcula_iva(258)

54.18

Ejecuta la función __calcula_Iva__ para las cantidades 145 y 258. Comprueba que los resultados son 30.45 y 54.18

#### 2 Ejercicio

Evalúa el polinomio $x^4 + x^3 + \frac{1}{2} x^2 - x$
en $x = 10$. El resultado
ha de ser  un número real.

In [12]:
# Sol:

# El resultado es 11040.0

In [13]:
evalua(7)

2761.5

Evalúa  el polinomio anterior para $x = 7$.

In [8]:
# Sol:

def evalua(x):
    p = x**4 + x**3 + 1/2*x**2 - x
    return(p)

In [9]:
evalua(7)

2761.5

#### 3 Ejercicio

Define una función llamada __raiz_cubica__ que devuelva el valor de $\sqrt[3]{x}$. Recuerda que  $\sqrt[3]{x}$ = ${x}^{\frac{1}{3}}$


In [1]:
# Sol:
# El resultado de raiz cúbica de 5 es: 1.7099759466766968

def raiz_cubica(x):
    s = x**(1/3)
    return(s)

In [2]:
raiz_cubica(5)

1.7099759466766968

In [3]:
valor = 5
'El resultado de la raíz %s de %d es: %.2f' % ('cubica', valor, raiz_cubica(5))

'El resultado de la raíz cubica de 5 es: 1.71'

Escribe la expresión Python que construya la siguiente cadena:

$$"El \ resultado \ de  \  la  \  raíz  \  cúbica  \   de  \ 5 \  es:  1.70"$$

Utiliza el operador de formato __%f__.

--------------------